docs/docs/PluginDevelopment.md

```diff
@@ -21,24 +20,31 @@ For example:
 ```bash
     dvp build
 ```
-    
-   4. Upload the `artifacts.json` ( generated in step 3 ) on Delphix Engine:
+
+
+Upload plugin
+----------------
+   Upload the `artifacts.json` ( generated in step 3 ) on Delphix Engine:
 ```bash
     dvp upload -e <Delphix_Engine_Name> -u <username> --password <password>
 ```
-   5. Unit test run: Make sure to build the source code( using `dvp build` ) before running unit tests
+
+
+Unit Test of Plugin code
+------------------------
+   Unit test run: Make sure to build the source code( using `dvp build` ) before running unit tests
   ```bash
      pytest test/
 ```
 
-### <a id="run_unit_test_case"></a>Download plugin logs
-#### Plugin Logs:
-Download the plugin logs using below command:
-
-```dvp download-logs -c plugin_config.yml -e <Delphix_Engine_Name> -u admin --password <password>```
-
-#### Unit test logs: 
 ##### SummaryReport:

```

Comment:
Do you think we could add an extra space -> "Summary Report"?

Reply:
Fixed